backend/app/tests/crud/evaluations/test_processing.py

```
@@ -0,0 +1,805 @@
+            "run_id": eval_run.id,
+            "run_name": eval_run.run_name,
+            "action": "no_change",
+        }
+
+        result = await poll_all_pending_evaluations(
+            session=db, org_id=test_dataset.organization_id
+        )
+
+        assert result["total"] == 1
+        assert result["still_processing"] == 1
+        mock_check.assert_called_once()
```

Ruff (0.14.13)

No newline at end of file

Add trailing newline

(W292)